freyja_apps/ibeji_adapter/README.md

```diff
@@ -0,0 +1,102 @@
+# Ibeji Adapter Freyja Example Application
+
+This Freyja Example Application utilizes the [Ibeji Digital Twin Adapter](../../freyja_adapters/digital_twin/ibeji_adapter/) and an in-memory mock cloud connector adapter to show a minimal connected local example of how to retrieve data from the vehicle.
+
+## Build and Run
+
+To build and run the application, follow these steps:
+
+1. (Optional) If necessary, author configuration overrides for the [`InMemoryMockMappingClient`](https://github.com/eclipse-ibeji/freyja/tree/main/mapping_clients/in_memory_mock_mapping_client). Refer to the adapter README files for instructions on how to do this. This repository provides overrides in the [`.freyja`](../../.freyja/) directory that can be used with the [`mixed` sample provided by Ibeji](https://github.com/eclipse-ibeji/ibeji/tree/main/samples/mixed).
+
+1. Set the `$FREYJA_HOME` environment variable. If you are using the provided overrides, you can run the following command to set the variable:
+
+        export FREYJA_HOME={path-to-repo-root}/.freyja
+
+    Alternatively, you can set the variable in a [Cargo configuration file](https://doc.rust-lang.org/cargo/reference/config.html) to only enable the variable while running a Cargo command.
+
+1. Run the following from the repo root:
+
+        cargo run -p freyja-ibeji-adapter-app
+
+    This will rebuild the application as necessary and then run it. Note that running Cargo commands without specifying the `-p` argument might target every package in the workspace! When working with this repository it's recommended to use the `-p` argument with Cargo commands.
```

Comment:
Please briefly describe what the -p option does.  That will help the reader better understand the implications. of not using it.

Reply:
the second sentence of this paragraph already does

Reply:
"Note that running Cargo commands without specifying the `-p` argument might target every package in the workspace!" describes the implication of not using it, not what is need for in the first place.

Reply:
I would argue that the first sentence does enough to describe what it does. the definition of -p in cargo is "By default, the package in the current working directory is selected. The -p flag can be used to choose a different package in a workspace"
https://doc.rust-lang.org/cargo/commands/cargo-run.html#package-selection

Reply:
Updated app readme's to be more concise. And in addition added some explanation for -p in the template's readme as that is a common spot. Please look again